src/app/theming/angular/angular-sample.component.scss

```diff
@@ -202,8 +204,31 @@ igx-expansion-panel-body {
     }
 
     ::ng-deep {
-        @include mat.core();
         @include theme($igx-light-palette);
         @include mat.all-component-themes($custom-mat-light-theme);
+
+        .mat-mdc-slider.secondary .mdc-slider__track--inactive,
+        .mat-mdc-slider.secondary .mdc-slider__thumb-knob {
+            background-color: color($custom-mat-light-theme, 'secondary');
+        }
+
+        .mat-mdc-slider.secondary .mdc-slider__track--active_fill,
+        .mat-mdc-slider.secondary .mdc-slider__thumb-knob {
+            border-color: color($custom-mat-light-theme, 'secondary');
+        }
+
+        .mat-mdc-slider.secondary .mat-mdc-slider-visual-thumb .mat-ripple > * {
+            background-color: color($custom-mat-light-theme, 'secondary');
+            opacity: .12;
+        }
+
+        .mat-stepper-horizontal.mb-1 .mat-horizontal-stepper-header-container {
+            margin-bottom: 16px;
+        }
+
+        .mat-mdc-form-field-focus-overlay,
+        .mdc-text-field--filled:not(.mdc-text-field--disabled) {
+            background-color: color($custom-mat-light-theme, 'surface');
+        }
```

Comment:
Do you do that because internally the mat slider uses the primary color for its elements, or because there isn't a way to implement this using another custom theme?

Reply:
Yes, the material slider uses primary color for its elements.